src/ar-AE.json

```diff
@@ -1274,7 +1274,9 @@
       "pasteEmbeddedCodeHere": "الصق التعليمات البرمجية المضمنة هنا",
       "emojiPickerTypeToFind": "اكتب للبحث",
       "emojiPickerNoResultFound": "لم يتم العثور على نتائج",
-      "emojiPickerTrySomethingElse": "جرب شيئًا آخر"
+      "emojiPickerTrySomethingElse": "جرب شيئًا آخر",
+      "linkAriaLabel": "افتح في نافذة جديدة",
+      "ImageLinkAriaLabel": "افتح في نافذة جديدة"
```

Comment:
you have not declared it in source, but you have declared it in ej2-locale. why?

Reply:
I have added the localization text for image and link in this PR, but seperately work for source level code.